<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search</class>
 <widget class="QWidget" name="search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>484</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1076</width>
    <height>484</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	/*background-color: rgba(255, 255, 255, 0.6);*/
	background-color: transparent;
	color: #fff8ea;
	border: 2px solid white;
	font-size: 16px;
	font-weight: 700;
	font-family: ZCOOL KuaiLe;
}
/*搜索框*/
#search_lineEdit{
	font-size: 20px;
	font-weight: 800;
	background: transparent;
	border: 2px solid white;
	border-radius: 6px;
	padding-left: 10px;
	padding-right: 10px;
}
#search_lineEdit:focus{
	border: 2px solid #E3629A;
	background: rgba(255, 255, 255, 0.2);
}
/*搜索按钮*/
#sub_search_btn{
	border-radius: 5px;
	border-image: url(:/play/tools/resource/bg/icon/play/search_1.png)；
}
#sub_search_btn:hover{
	/*border-radius: 5px;
	background-color: rgba(255, 255, 255, 0.4);*/
	margin: 2px;
}
/*搜索按钮*/



/*垂直滚动条样式*/
QScrollBar::add-line, QScrollBar::sub-line {
    width: 0;  /* 隐藏水平滚动条的箭头 */
    height: 0; /* 隐藏垂直滚动条的箭头 */
}
QScrollBar::add-page, QScrollBar::sub-page {
    background: transparent;
    /*background: rgba(255, 255, 255, 0.5);*/
	border: 0px;
	border-radius: 2px;
	padding-top: 2px;
    padding-bottom: 2px;
}
QScrollBar{
    width: 10px;
    border: 0px;
    border-radius: 5px;
	padding: 2px;
}
QScrollBar::handle{
	width: 4px;
	border-radius: 2px;
	/*background-color: #84ACD9;*/
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #639AD4, stop: 1 #DE76A3);
}
QScrollBar::handle:hover{
	/*background-color: #75ACEB;*/
	background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0, stop: 0 #639AD4, stop: 1 #DE76A3);
}
QScrollBar:vertical {
    border: none;          /* 去除滚动条整体边框 */
    background: transparent;  /* 背景透明（可选） */
}
QScrollArea .QWidget{
	border: none;
}
QScrollArea{
	padding-top: 5px;
	padding-bottom: 5px;
}
/*垂直滚动条样式*/

/*推荐页面通用样式*/
[recommend]{
	font-weight: 700;
	font-size: 30px;
}
[day]{
	font-size: 18px;
}
/*推荐页面通用样式*/</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_25">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_36">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="search_lineEdit">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sub_search_btn">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_37">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="search_music_scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1032</width>
        <height>393</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_29">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="search_music_verticalLayout"/>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>313</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
